root/etc/s6-overlay/s6-rc.d/init-mod-code-server-python3-poetry-install/run

```diff
@@ -4,5 +4,12 @@
 # so it can take advantage of packages installed
 # init-mods-end depends on this script so that later init and services wait until this script exits
 
-echo "**** Setting up apprise ****"
-apprise blah blah
+echo "**** Setting up poetry ****"
+
+# Install poetry
+curl -sSL https://install.python-poetry.org | python3 -
+
+# Add poetry to the path
+echo "export PATH=\"\$HOME/.local/bin:\$PATH\"" >> /root/.bashrc
```

Comment:
The path should be updated like in here: https://github.com/linuxserver/docker-mods/blob/code-server-golang/root/etc/s6-overlay/s6-rc.d/init-mod-code-server-golang-add-package/run#L3-L9
depending on whether it needs to go in the beginning or the end